src/Cli/dotnet/Commands/Package/List/LocalizableStrings.resx

```diff
@@ -174,4 +174,11 @@
   <data name="CmdOutputVersionDescription" xml:space="preserve">
     <value>Specifies the version of machine-readable output. Requires the '--format json' option.</value>
   </data>
+  <data name="Error_restore" xml:space="preserve">
+    <value>Restore failed. Run `dotnet restore` for more details on the issue.</value>
+    <comment>Do not translate `dotnet restore`</comment>
```

Comment:
nit: use the [locking syntax](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/Localization.md#locking-translations) for pinned translation strings so that the Loc team's tooling helps prevent unintentional translation.